x-pack/libbeat/management/api/configuration.go

```diff
@@ -9,13 +9,17 @@ import (
 	"net/http"
 	"reflect"
 
+	"github.com/pkg/errors"
```

Comment:
did you mean package "errors" or package "github.com/pkg/errors" :)

Reply:
I did mean `errors`, not sure why goimports doesn't prioritize std library, I will pay more attention to that.